ITEM.tsv

```diff
@@ -4175,8 +4175,8 @@ ITEM_20150323_004174	Priest Roana's Research
 ITEM_20150323_004175	A research report written by Priest Roana. It contains research on how curses are made. 
 ITEM_20150323_004176	Priest Prozit's Research 
 ITEM_20150323_004177	A research report written by Priest Prozit. It contains research on the effects of curses. 
-ITEM_20150323_004178	A relic that is treasured by the Church. Unfortunately it's been discarded across the Cathedral. 
-ITEM_20150323_004179	An orb that can detect holy power. It can help the user find the Graceful Relic. 
+xITEM_20150323_004178	A relic that is treasured by the Church. Unfortunately have been discarded across the Cathedral. 
```

Comment:
Shouldn't it be "has" instead of "have"? The first sentence is singular. And it seems the subject is still missing for the second sentence. If you don't like the "it", a suggestion would be to mix these sentences:

A relic that was once treasured by the Church has, unfortunately, been discarded across the Cathedral.

I'm assuming the Church/Cathedral are organizations, not places. Still, the change feels pretty minor imo? (just sentence structure)